src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1858,6 +1858,7 @@ internal enum ErrorCode
         ERR_DoesNotOverrideMethodFromObject = 8869,
         ERR_SealedGetHashCodeInRecord = 8870,
         ERR_DoesNotOverrideBaseEquals = 8871,
+        ERR_CloneDisallowedInRecord = 8872,
```

Comment:
>8872 [](start = 38, length = 4)

There is already a PR under review that uses numbers through 8874. Consider picking one of the next numbers and also leaving an empty line above this member to simplify merging process.